src/reports.py

```diff
@@ -1,9 +1,14 @@
 import datetime
+import json
 import os
 
 import jinja2
 
-from src.data_fetcher import DataFetcher
+
+def load_data(filename):
+    with open(filename, 'r') as f:
+        json_data = f.read()
+    return json.loads(json_data)
```

Comment:
Let's import the new `stats.StatsAggregator` class, create an instance of it, and use its `_load_data()` method.